packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -1557,7 +1557,8 @@ export function traverseEnrichedBlocks(
                     "sdg-grid",
                     "sdg-toc",
                     "topic-page-intro",
-                    "all-charts"
+                    "all-charts",
+                    "align"
```

Comment:
This should get its own callback that iterates through `content` so that `Gdoc.links` and `Gdoc.filenames` will work

Reply:
I see, thank you!